apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js

```diff
@@ -41,6 +51,14 @@ registerPlugin( 'wpcom-block-editor-nux', {
 			return null;
 		}
 
+		// Open patterns panel before Welcome Tour if necessary (e.g. when using Blank Canvas theme)
+		// Do this only when Welcome Tour is not manually opened.
+		if ( variant === 'blank-canvas-tour' && ! isManuallyOpened ) {
```

Comment:
I think this logic is causing the page pattern modal to open **twice** if the user doesn't follow the site setup checklist.

1. Create a new site at `/start/with-design-picker`
2. Choose blank canvas
3. After landing on My Home, **don't choose edit homepage from checklist**
4. Create a new page another way e.g. go to `/pages` list and click "Add new page"
5. Page pattern modal opens as expected, make a selection
6. **Notice that the page pattern modal opens a second time**

I think maybe the issue is that there's an assumption that the first call to `/block-editor/nux` is going to be for the blank homepage, but that's not necessarily the case.